src/Microsoft.IdentityModel.Tokens/JsonWebKeySet.cs

```diff
@@ -117,73 +139,150 @@ public JsonWebKeySet(string json, JsonSerializerSettings jsonSerializerSettings)
         [JsonProperty(DefaultValueHandling = DefaultValueHandling.Ignore, NullValueHandling = NullValueHandling.Ignore, PropertyName = JsonWebKeySetParameterNames.Keys, Required = Required.Default)]
         public IList<JsonWebKey> Keys { get; private set; } = new List<JsonWebKey>();
 
+        /// <summary>
+        /// Flag that controls whether unresolved JsonWebKeys will be included in the resulting collection of <see cref="GetSigningKeys"/> method.
+        /// </summary>
+        [DefaultValue(true)]
+        public static bool SkipUnresolvedJsonWebKeys { get; set; } = true;
```

Comment:
I opened this to capture perhaps have a way to set this at runtime.
https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/issues/1165

Reply:
Do we still want to keep the flag for this PR?